src/Security/Security/Cmdlets/SqlVulnerabilityAssessment/AddSqlVulnerabilityAssessmentBaseline.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Commands.Security;
+using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Cmdlets.SqlVulnerabilityAssessment
+{
+    [Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVulnerabilityAssessmentBaseline"),OutputType(typeof(PSSqlVulnerabilityAssessmentBaselineResults))]
```

Comment:
Please set a default parameter set for this cmdlet.

Reply:
pending